README.md

```diff
@@ -1,70 +1,44 @@
-# Getting Started with Create React App
+![Logo](./public/assets/android-chrome-192x192.png)
```

Comment:
This entire Readme looks so good, Zach! I love the link to another file for running the app locally.